doc/content.tex

```diff
@@ -1,1347 +0,0 @@
-
-    There are also seasonal variants of these models which holds one model for each season. These methods in general
-     produces high forecasting error but are very easy to implement.
-
-    %%%%%%%%
-    \section{Linear Regression} \label{sec:linear-regression}
-    Linear regression is a classical statistical analysis technique. It is mostly used to determine whether there is
-    linear relationship between dependent and eventually more independent variables \cite{cipra}. It is also used for
-    predictions mainly in econometric field.
-
-    Simple linear regression is defined as:
-
-    \begin{eqnarray} \label{eq:linear-regression}
-        y = \beta_0 + \beta_1 x + \epsilon
-    \end{eqnarray}
-
-    Parameters $\beta_0$ and $\beta_1$ are calculated by minimizing the sum of squared errors:
-    
-    \begin{eqnarray} \label{eq:linear-regression-estimation}
-        SSE = \sum_{i=1}^N \epsilon_{i}^2 = \sum_{i=1}^N (y_i - \beta_0 - \beta_1 x_i)^2
-    \end{eqnarray}
-    
-    Once the parameters are estimated predictions for any time in the future can be calculated. If modelled time
-    series is not stationary and trend changes over time parameters should be periodically reestimated to achieve
-    better forecasting accuracy. So the regression model does not adapt if the underlying time series changes.
-
-    %For econometric analysis it is important to use best linear unbiased estimator,
-    %where couple of assumptions has to hold:
-    %\begin{enumerate}
-    %    \item $E(Y_i) = \beta_0 + \beta_1 X_i$
-    %    \item $var(Y_i)=\sigma^2$\,--\,homoscedasticity
-    %    \item $cov(Y_i, Y_j)=0$, for $i \neq j$
-    %    \item $Y_i$ cames from normal distribution
-    %    \item $X_i$ is not random variable
-    %\end{enumerate}
-    %Where $i$ takes values from $1$ to the number of observed values. These assumptions 
-
-    %%%%%%%%
-    \section{Simple Exponential Smoothing} \label{sec:simple-ex}
-    The concept behind simple exponential smoothing is to attach larger weights to the most recent observations than
-    to the observations from distant past. Forecasts are calculated using weighted averages where the weights
-    decrease exponentially as observations come from further in the past \cite{hyndman-state-space}. In other words
-    smaller weights are associated to older observations. Equation for simple exponential smoothing is listed in
-    \ref{eq:simple-ex}.
-
-    \begin{gather} \label{eq:simple-ex}
-         \hat{y}_{T+1|T} = l_t \\ \nonumber
-         l_t = \alpha y_t + (1-\alpha)l_{t-1}
-    \end{gather}
-
-    For smoothing parameter $\alpha$ holds $ 0 \leq \alpha \leq 1 $. Note, if $\alpha = 1$ then \\
-    $\hat{y}_{T+1|T} = y_{T}$ so forecasts are equal to the na\"{i}ve method. If the parameter $\alpha $ is smaller
-    more weight is given to the observations from distance in the past.
-
-    Simple exponential smoothing has a flat forecast function, that means all forecasts all the same.
```

Comment:
that means all forecasts all the same - tu nie som si ista, nema byt to druhe "all" nahradene "are"?